### PR TITLE
Change 'friend' to 'contact'

### PR DESCRIPTION
**Pull request by Tirifto**
Change 'friend' to 'contact'
=====================

**Summary:**

+ In Tox, contacts (Tox IDs) the user has added should be called 'contacts' rather than 'friends'. This has been discussed briefly in Tox/Tox-STS Issue 3, and more extensively in tux3/qTox Issue 1855. Following arguments should mostly come from there.

**Arguments for:**

These are arguments for the proposal:
+ The word 'friend' is unfitting and misleading.
  + The contacts do not have to be user's friends at all. They can be coworkers, classmates, relatives, or anyone else. With the wide variety of every person's own relationships and possible usages of Tox, calling contacts friends is improper.
  + The word 'contact' is accurate. All of the people on your list may not be your friends, but they will always be your contacts, since Tox can be used to contact them. It addresses all types of relationships between people.
  + The word 'friend' would be even more unfitting if Tox was used in work as a tool for business.
+ The word 'friend' would be confusing in contact organizing.
  + If a Tox client implements organizing contacts into groups/aspects/circles, the user might like to make a group for their actual friends, and another groups for other people. Calling every contact a friend would be confusing and/or strange in this case.
+ The word 'contact' is widely used and recognized in instant messaging, social networking and other services. Everyone understands its meaning, even in the computer world, so it will cause no confusion and misunderstandings.

**Arguments against:**

These are arguments against the proposal:
- 'Contact' is a very bad word and it's not good for UX. (Elaborations welcome.)
- Facebook uses 'friend', too, so a lot of people are used to it.
- 'Friend' is more familiar and comfortable for the user.

**Status quo:**

At the moment, 'friend' is used in most Tox clients and the STS. There were attempts to change this, but they never managed to do so. Now that the STS has been revived, we have a way to make this change possible.

Five Eyepatches, please take this pull request in consideration and share your opinions.

@dvor @subliun @tux3 @Impyy @irungentoo

### CODE REVIEW

**Comment by tux3** [comment]
I'm leaning towards contact instead of friend, however there seemed to be a fairly large pushback against it, so I'd like to withhold my vote until there's a bit more discussion.

It's clear to me that every contact isn't necessarily a friend, and "friend" could be interpreted pretty strongly in some languages. But I can see how the word doesn't have that much meaning, especially with how Facebook uses it.

**Comment by ProMcTagonist** [comment]
What does Facebook use in other languages?

**Comment by Tirifto** [comment]
> What does Facebook use in other languages?

In Czech language, Facebook term for 'friends' is 'přátelé'. I don't remember Facebook that well, but I deduce it was always used in plural. In singular the word is 'přítel'/'přítelkyně', which can also mean 'boyfriend'/'girlfriend', so that would be awkward. Another alternative is 'kamarád', but that is informal and sounds strange.

Facebook is using the word lightly and I think Tox shouldn't follow that example. I believe we should take the meaning of words in consideration and use them appropriately. Popular usage plays a role, too, but shouldn't be that major here.

**Comment by zetok** [comment]
as for proposal
>Include in commit and pull request description what change is about

commit doesn't have needed stuff included `:/`

====

As for actual content; drawbacks:
>* 'Contact' is a very bad word and it's not good for UX.
>* 'Friend' is more familiar and comfortable for the user.

Wording is very important when it comes to how people perceive things. `friend` even in English doesn't have to mean real friend - e.g. saying ``Hello, my friend, what's up?`` doesn't mean that people are friends to each other, but rather a casual interaction.

That border is being further blurred by social media using it as they please, sometimes using it beyond its usual meaning, but rarely.


1. Why using it in Tox is important?
2. Why replacing it is wrong?

.

1. Wording, as many other things, is being used to make people like Tox - no matter how you look at it, word `friend` usually has a "warm feeling" attached to it - using it in Tox makes people unconditionally connect this warm feeling with Tox. ~same reason why fb uses it - people think warmer about something that provides them with means of communicating with "friends", even if in reality they're not "friends" so much. You can't escape what your subconsciousness tells you, something that was engraved on you since early childhood. Thus, to make Tox as popular as possible, wording should be carefully selected - in this case, `friend` was ~proven by fb to be a "good word" that people will generally ~accept, and that will help people think about something that uses this word in better light.
2. Removing all of stuff mentioned in 2. will be really bad for Tox, since one of things that make it better will be removed. Even in a case where it would be replaced with a word of an equivalent "warmth", could we really be sure that it will be accepted by people? That they will endorse and enjoy using it? But this isn't the case. This proposal is about replacing it with cold, de-humanized, emotionally empty word, a ``contact``. Someone (something actually) that you exchange data with. Not really joke around, share time and thoughts, but simply give and receive data, whatever should be given / received, according to strict set of rules.

----

As for points raised above: how exactly by removing good stuff from Tox it would become better? 

====
>What does Facebook use in other languages?

In Polish fb uses ``znajomi``, which is ~equivalent in meaning to "someone you know". Feels very natural.

====
@Tirifto 
>In Czech language, Facebook term for 'friends' is 'přátelé'. I don't remember Facebook that well, but I deduce it was always used in plural. In singular the word is 'přítel'/'přítelkyně', which can also mean 'boyfriend'/'girlfriend', so that would be awkward. Another alternative is 'kamarád', but that is informal and sounds strange.

From what I understood, your main issue isn't that it's a word `friend`, but about how fb translates it, and that qTox translation guidelines encourage you to use whatever fb uses?

Eh. Whatever is written there, is written because I thought that it'll be a good idea to follow whatever fb does when it comes to translating `friend` - since fb used in Polish version word that ~naturally[1] fit into it, I thought that this would be also the case in other languages. If that's not the case, then instead of following what fb does or exact translation, you should translate it to whatever feels the best, while preserving "warmth" of concept - i.e. translate it to something that'll have meaning of ``friend`` and/or ``someone you know``, whatever meaning feels better.

[1] - As naturally as fb's original "friend"

**Comment by Tirifto** [comment]
@zetok 

> From what I understood, your main issue isn't that it's a word friend, but about how fb translates it, and that qTox translation guidelines encourage you to use whatever fb uses?

Both of them are my main issues here.

Concerning the 'warmth', I see what you mean. You are probably right, although I personally (and maybe other people, athought it might be a minority) don't feel better about Facebook and other services which use the term 'friend'.

While the word 'friend' might make the service more pleasant, warm and familiar for some, I feel like the services who use that word misrepresent my relationships and force me to use something I use for people I care for, for everyone else, too. While it may have good effects, it can also bear some bad effects.

Some people I know (and perhaps some in media) have also criticized this trend, since people nowadays make friendships that are only symbolic, over the internet, and they don't hold any value. While Facebook might have brought positive subconscious reactions, it also brought negative conscious reactions.

Personally, I'd prefer focusing more on the conscious mind. Doing things for their subconscious effect feels like playing with the users and trying to twist their view on reality – but that might be just me.

> This proposal is about replacing it with cold, de-humanized, emotionally empty word, a contact.

I still think the neutrality and other benefits 'contact' brings are more important and valuable (and concrete), than the warmth and subconscious benefits of 'friend'. Furthermore, the warmth will come from the person, if you have a warm relationship. If you have a cold relationship, 'friend' might do more bad than good to one's subconsciousness. 'Contact' will be suitable for anyone.

> In Polish fb uses znajomi, which is ~equivalent in meaning to "someone you know". Feels very natural.

I think that "someone you know" is very good and brings a good combination of neutrality and warmth. Czech language has this word, too (známí). The closest English word I can think of is 'acquaintance'. As a non-native speaker, I found the word to be a bit... complicated. But if we assume that a number of users will have their native language as a choice, perhaps it would come in consideration?

> commit doesn't have needed stuff included :/

How about we evaluate this PR, and if it gets approved by the votes, I will make new one, identical to this one, except that it will have the commit fixed and this PR will be referenced in order to keep the discussion and voting easily accessible. Would that work?

**Comment by Impyy** [comment]
I agree with this.
'Contact' is more correct and Skype is using it too, which is what we're trying to replace after all.

**Comment by subliun** [comment]
I agree with this. Antox uses 'friend' internally to distinguish between friends, groups and contacts (which can be either groups or friends). I have no problem with changing all user-facing strings that say 'friend' to contact though, it sounds more professional.

**Comment by GrayHatter** [comment]
TL;DR, Use contact, and then define the words we want to use.

We should define how we expect these words to be used in the STS... As far as friend or contact. We shouldn't use friend, to almost echo @Impyy Contact is what google uses, and they're crazy detailed about grammar. 

IMO the STS should be dry, and not concerned with how the user interacts with the client. That's up to how the client wants to work. The STS should use contact, because that's what we can be sure they are. Someone/thing a user wants to make contact with. A was going to suggest peer, but a peer is anyone who clients can send data to or through, in an attempt to communicate with a contact. Finally all friends are contacts, but not all contacts are friends.

**Comment by dvor** [comment]
I vote for using `contact`.

After reading all arguments I think that 'professional' contact bring more benefit that 'warmth' of friend.

**Comment by zetok** [comment]
@tux3 for the record, after more arguments you still vote for that change?

@Tirifto new PR?

**Comment by tux3** [comment]
@zetok I'd like to withhold my vote, I don't have a strong opinion on this.

**Comment by zetok** [comment]
Not merging, since those changes are simply counter-productive.

toxcore heavily uses `friend`, and changing things in STS would do more harm than good (confusion, etc) to developers who would try to figure out for the first time how things are.

With that being said, some sort of separation to how underlying stuff would work and what properties UI should have would be welcomed in STS.

**Comment by aaannndddyyy** [comment]
+1 on using "contact"
